bundles/org.openhab.binding.entsoe/src/main/resources/OH-INF/i18n/entsoe.properties

```
@@ -0,0 +1,135 @@
+# add-on
+
+addon.entsoe.name = ENTSO-E Binding
+addon.entsoe.description = This is the binding for European Network of Transmission System Operators (ENTSO-E) to receive electricity market spot prices.
+
+# thing types
+
+thing-type.entsoe.day-ahead.label = Day ahead prices
```

Regenerate file if needed, or if this is the only change since last time, at least this text must be updated:
```suggestion
thing-type.entsoe.day-ahead.label = Day-ahead prices
```